openfisca_us/parameters/irs/credits/child_and_dep_care/phaseout/rate.yaml

```diff
@@ -0,0 +1,5 @@
+description: Child & dependent care credit phaseout percentage rate
+metadata:
+  unit: /1
+values:
+  2013-01-01: 0.0005
```

Comment:
This value was confusing until I read the note in the taxcalc description, could you add that here? http://taxcalc.pslmodels.org/guide/policy_params.html#cdcc-prt

> In the law, the credit rate is reduced by 1 percentage point for every $2,000 of AGI over the limit.